public/jsx/components/card-choice.jsx

```diff
@@ -2,12 +2,13 @@
 
 var $ = require('jquery');
 var React = require('react');
-var Navigation = require('react-router').Navigation;
 
 var CardItem = require('components/card-item');
 var SubmitButton = require('components/submit-button');
 
 var gettext = require('utils').gettext;
+var purchaseActions = require('purchase-actions');
```

Comment:
Would it make sense to put all actions in their own dir? So this could be `require('actions/purchase')` for example.